app/views/passwords/edit.html.erb

```diff
@@ -8,6 +8,10 @@
     <%= form.label :password, "Password", :class => 'form__label' %>
     <%= form.password_field :password, :class => 'form__input' %>
   </div>
+  <div class="form__checkbox">
+    <%= form.check_box :reset_api_key, :class => 'form__checkbox__input' %>
```

Comment:
Let's pin form value here and check for just that in the controller.
```
form.check_box :reset_api_key, { :class => 'form__checkbox__input' } , 'true', 'false'
```

Reply:
Done in https://github.com/spk/rubygems.org/commit/2007dc6a51be9aa2f0805aa868333655385171a8